---
bool return_navi_ok
